core/src/test/java/org/apache/iceberg/TestSnapshotSummary.java

```diff
@@ -358,4 +358,66 @@ public void rewriteWithDeletesAndDuplicates() {
         .containsEntry(SnapshotSummary.TOTAL_FILE_SIZE_PROP, "20")
         .containsEntry(SnapshotSummary.TOTAL_RECORDS_PROP, "1");
   }
+
+  @TestTemplate
+  public void testFileSizeSummaryWithDVs() {
+    assumeThat(formatVersion).isGreaterThanOrEqualTo(3);
+
+    DeleteFile dv1 = newDV(FILE_A);
+    table.newRowDelta().addDeletes(dv1).commit();
+
+    DeleteFile dv2 = newDV(FILE_B);
+    table.newRowDelta().addDeletes(dv2).commit();
+
+    Map<String, String> summary1 = table.currentSnapshot().summary();
+    long addedPosDeletes1 = dv2.recordCount();
+    long addedFileSize1 = dv2.contentSizeInBytes();
+    long totalPosDeletes1 = dv1.recordCount() + dv2.recordCount();
+    long totalFileSize1 = dv1.contentSizeInBytes() + dv2.contentSizeInBytes();
+    assertThat(summary1)
+        .doesNotContainKey(SnapshotSummary.ADD_POS_DELETE_FILES_PROP)
```

Comment:
minor: other tests in this class do a `hasSize()` check and check for all props (except for the `iceberg-version`). This is being done mainly to make sure the summary contains all the expected info.
I think it would be good here to add the `hasSize(12)` check and check that all of the 11 (not counting `iceberg-version`) have the right values